lib/ansible/utils/module_docs_fragments/nso.py

```diff
@@ -32,4 +32,8 @@ class ModuleDocFragment(object):
   password:
     description: NSO password
     required: true
+  timeout:
+    description: JSON-RPC request timeout in seconds
+    type: int
```

Comment:
Only type that is supported is bool, so this line can be removed.

Please add `version_added: "2.6"`

Reply:
Thanks, fixed now.